gui/slick/views/layouts/main.mako

```diff
@@ -283,51 +281,25 @@
         <footer>
             <div class="footer clearfix">
             <%
-                myDB = db.DBConnection()
-                today = str(datetime.date.today().toordinal())
-                status_quality = '(%s)' % ','.join([str(quality) for quality in Quality.SNATCHED + Quality.SNATCHED_PROPER])
-                status_download = '(%s)' % ','.join([str(quality) for quality in Quality.DOWNLOADED + Quality.ARCHIVED])
-
-                sql_statement = 'SELECT ' \
-                + '(SELECT COUNT(*) FROM tv_episodes WHERE season > 0 AND episode > 0 AND airdate > 1 AND status IN %s) AS ep_snatched, ' % status_quality \
-                + '(SELECT COUNT(*) FROM tv_episodes WHERE season > 0 AND episode > 0 AND airdate > 1 AND status IN %s) AS ep_downloaded, ' % status_download \
-                + '(SELECT COUNT(*) FROM tv_episodes WHERE season > 0 AND episode > 0 AND airdate > 1 AND ((airdate <= %s AND (status = %s OR status = %s)) ' % (today, str(SKIPPED), str(WANTED)) \
-                + ' OR (status IN %s) OR (status IN %s))) AS ep_total FROM tv_episodes tv_eps LIMIT 1' % (status_quality, status_download)
-
-                sql_result = myDB.select(sql_statement)
-
-                shows_total = len(sickbeard.showList)
-                shows_active = len([show for show in sickbeard.showList if show.paused == 0 and show.status == "Continuing"])
-
-                if sql_result:
-                    ep_snatched = sql_result[0]['ep_snatched']
-                    ep_downloaded = sql_result[0]['ep_downloaded']
-                    ep_total = sql_result[0]['ep_total']
-                else:
-                    ep_snatched = 0
-                    ep_downloaded = 0
-                    ep_total = 0
-
+                stats = Show.overall_stats()
+                ep_downloaded = stats['episodes']['downloaded']
+                ep_snatched = stats['episodes']['snatched']
+                ep_total = stats['episodes']['total']
                 ep_percentage = '' if ep_total == 0 else '(<span class="footerhighlight">%s%%</span>)' % re.sub(r'(\d+)(\.\d)\d+', r'\1\2', str((float(ep_downloaded)/float(ep_total))*100))
 
                 try:
                     localRoot = srRoot
```

Comment:
I have no clue, it was from before we went to mako so I think it was just old code that I hadn't gotten around to removing. I want to run a linter over the mako files for things like that but I can only find normal HTML linters. :disappointed: